app/models/solidus_paypal_braintree/response.rb

```diff
@@ -18,18 +19,14 @@ def build(result)
       def build_success(result)
         transaction = result.transaction
 
-        test = true
-        authorization = transaction.id
-        fraud_review = nil
-        avs_result = nil
-        cvv_result = nil
-
         options = {
-          test: test,
-          authorization: authorization,
-          fraud_review: fraud_review,
-          avs_result: avs_result,
-          cvv_result: cvv_result
+          test: !Rails.env.production?,
```

Comment:
Rather than `true` or `!Rails.env.production`, this should either be `nil` (indicating we have no idea) or something braintree tells us about using a sandbox.